dali/pipeline/util/copy_with_stride.cu

```diff
@@ -181,7 +181,7 @@ DALI_DEVICE DALI_FORCEINLINE void AlignedCopy(const StridedCopyDesc &sample,
                                               MismatchedNdimT mismatched_ndim) {
   using T = typename ElementTypeDesc::type;
   using VecT = typename ElementTypeDesc::vec_type;
-  constexpr int vec_len = ElementTypeDesc::vec_len;
+constexpr int64_t vec_len = ElementTypeDesc::vec_len;
```

Comment:
It's later used in expressions which could overflow with 32bit types

Reply:
indent